examples/create_tet_mesh.py

```diff
@@ -0,0 +1,27 @@
+import gustaf as gus
+
+
+def main():
+
+    mesh_faces_triangle_bs = gus.create.faces.box(
```

Comment:
I think this example can be merged into the other example with some comments

Reply:
so one example file ```create_mesh.py``` ?

Reply:
`create_boxes.py`?